packages/core/package.json

```diff
@@ -13,7 +13,7 @@
   },
   "devDependencies": {
     "@awaitjs/express": "^0.3.0",
-    "@eth-optimism/contracts": "^0.4.13",
+    "@eth-optimism/contracts": "^0.5.2",
```

Comment:
this version bump enables us to pull the compatible contracts directly from their exports.